### PR TITLE
Add scss-lint

### PR DESCRIPTION
**Pull request by timurvafin**
@rustemginiyatullin could you please continue in this branch and cleanup `scss-lint` warnings, please?

### CODE REVIEW

**Comment by rustemginiyatullin** [comment]
@timurvafin sure

**Comment by vast** [comment]
@rustemginiyatullin, @timurvafin, please take a look

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- Add scss-lint for reporting violations of SCSS coding conventions
```
Comment (timurvafin) [inline CHANGELOG.md]: Please add link to scss-lint github repo.

**Comment by timurvafin** [comment]
@vast please squash/organize commits.

**Comment by vast** [comment]
closing in favor of #280

**Pull request closed without merging**